snuba/admin/iam_policy/iam_policy.json

```diff
@@ -14,7 +14,8 @@
     },
     {
       "members": [
-        "group:team-sns@sentry.io"
+        "group:team-sns@sentry.io",
+        "group:access-snuba-admin@sentry.io"
```

Comment:
@volokluev I've added them to the `MigrationsReader` role. A user must have the `NonBlockingMigrationsExecutor` or `AllMigrationsExecutor` to execute a migration.